src/games/strategy/engine/framework/mapDownload/MapListingSource.java

```diff
@@ -20,31 +14,13 @@
  * Can be used to create a <code>MapDownloadAction</code>
  */
 public class MapListingSource {
-
-  public static final String MAP_LIST_DOWNLOAD_SITE_PROPERTY_KEY = "Map_List_File";
   private final String mapListDownloadSite;
 
-
-  public MapListingSource(File mapDownloadPropertiesFile) {
-    checkState(checkNotNull(mapDownloadPropertiesFile).isFile());
-    mapListDownloadSite = readMapListDownloadSitePropertyValue(mapDownloadPropertiesFile);
+  public MapListingSource(PropertyReader propertyReader) {
```

Comment:
Same for MapListingSource, it got a ton simpler now that it does not need to do  file parsing.